build/build.sh

```diff
@@ -10,15 +10,15 @@ set -ex
 main() {
     CARGO="$(builder)"
 
-    if is_linux && is_musl; then
-        # jemalloc doesn't allow 16K page sizes for linux musl 
-        # https://github.com/microsoft/ripgrep-prebuilt/issues/26
-        JEMALLOC_SYS_WITH_LG_PAGE=14
-    fi
-
     # Test a normal debug build.
     if is_arm || is_aarch64 || is_ppc64le; then
-        "$CARGO" build --target "$TARGET" --release --features 'pcre2'
+        if is_linux && is_aarch64 && is_musl; then
```

Comment:
```suggestion
        if is_linux && is_aarch64_musl; then
```

Reply:
sorry this should be changed, refs https://github.com/microsoft/ripgrep-prebuilt/blob/aa3a29dd462cd264dd4d5a302b8309660fc3df97/build/utils.sh#L108C1-L113 , `is_aarch64` is only returned for the gnu build.